src/types.ts

```diff
@@ -48,222 +111,228 @@ export interface Metadata {
  * @example
  * {
     "_id": "7406ab7d9e781685d6d0ceb319b84b332ff1b773ff0bbce1671d843d50c9532a",
-    "created_at": new Date(1645796112916),
+    "createdAt": new Date(1645796112916),
     "metadata": [
       {
         "k": "user_agent",
         "v": "SpvWalletClient v1.0.0"
       },
     ],
-    "current_balance": 99848517,
-    "next_internal_num": 100,
-    "next_external_num": 229
+    "currentBalance": 99848517,
+    "nextInternalNum": 100,
+    "nextExternalNum": 229
   }
  */
-export interface XPub {
+export interface User {
+  metadata?: Metadata;
+  id: string;
+  currentBalance: number;
+  nextInternalNum: number;
+  nextExternalNum: number;
+  createdAt?: Date;
+  updatedAt?: Date;
+  deletedAt?: Date;
+}
+
+/**
+ * Admin stats interface
+ */
+export interface AdminStats {
   /**
-   * metadata object
+   * Total balance of all outputs in sats in the database
    */
-  metadata?: Metadata;
+  balance: number;
   /**
-   * xpub id
+   * Number of destinations in the database
    */
-  id: string;
+  destinations: number;
   /**
-   * Current balance in sats of the xpub
+   * Number of transactions in the database
    */
-  current_balance: number;
+  transactions: number;
   /**
-   * Next internal (change address) number to use for a new destination
-   *
-   * NOTE: Do not use this to create new destinations, always let SPV Wallet create the destination
+   * Number of paymail addresses in the database
    */
-  next_internal_num: number;
+  paymail_addresses: number;
   /**
-   * Next external number to use for a new destination
-   *
-   * NOTE: Do not use this to create new destinations, always let SPV Wallet create the destination
+   * Number of utxos in the database
    */
-  next_external_num: number;
+  utxos: number;
   /**
-   * Date when this object was created
+   * Number of xpubs registered in the database
    */
-  created_at?: Date;
+  xpubs: number;
   /**
-   * Date when this object was last updated
+   * A key value object of dates and number of transactions on that date (YYYYMMDD)
    */
-  updated_at?: Date;
+  transactions_per_day: { [key: string]: any };
```

Comment:
Perhaps a number would be more appropriate.
```suggestion
  transactions_per_day: { [key: string]: number };
```

Reply:
![image](https://github.com/user-attachments/assets/dce48465-5905-41c9-8822-b21dbf2f9c32)
I would like to match the interface from spv-wallet which is string/any so I wouldn't like to change this field